packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/FormSelectDemo/FormSelectDemo.tsx

```diff
@@ -52,11 +52,11 @@ export class FormSelectDemo extends Component<{}, FormSelectState> {
     ];
   }
 
-  onChange = (value: string, event: any) => {
+  onChange = (value: string) => {
     this.setState({ value });
   };
 
-  onChangeValidatedSelect = (value: string, event: any) => {
+  onChangeValidatedSelect = (value: string) => {
```

Comment:
As a demo, it would be good to show what args can be are expected here

Reply:
Added argument back in